free-3.md

```diff
@@ -10,59 +10,116 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea has to do with asynchronous tasks. It's not super well fleshed out,
+but I know it will be something similar to cron, but making it very
+easy to automate tasks.
 
-### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+I've never used cron myself (I would definitely try it out before I try to
+make a new version or anything), but I've heard that it can be a bit
+clunky and not super intuitive to learn.  Hopefully anyone who needs to
+fire off tasks to run at intervals would be able to use a more friendly
+software to do so.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
-
+I would be excited for the results of the language, being able to just
+write a quick description of what to run, how, and when, and have it
+just run when it should.  I would also be excited to learn a bit more
+about how cron works and the best way to abstract cron functionality
+into abstract language concepts.
 ### Domain
 _Describe the project's domain in five words._
 
+Assign tasks asynchronously with delays.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I would like to support at least as much functionality as cron,
+that is, to run some executable or process periodically, but also
+some ability to do conditional operations:
+```
+if /dev/usb0 exists:
+  do once:
+    echo 'A usb device was plugged in'
+```
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When a program runs, it should create a cron job that fulfills the given
+requirements.  It could also run a background process that periodically
+queries the state of the computer (if conditional jobs were present in
+the input program) to see if a new process needed to be created for
+the given job.
+
+Errors might involve failure to start at a specified time, due to
+not having processor time at that moment, or any number of file system or
+operating system failures.  In any case, the error should probably be
+best logged to a file *and* reported to the user via a shell notification.
```

Comment:
I like the 2 methods for notifying user.  Seems important to make sure user knows.